pkg/repository/storage/postgresql/connection.go

```diff
@@ -44,7 +44,7 @@ func connectToDb(host, user, password, dbname, port, sslmode, timezone string, l
 		port = detectedPort
 	}
 
-	dsn := fmt.Sprintf("host=%v user=%v password=%v dbname=%v port=%v sslmode=%v TimeZone=%v", host, user, password, dbname, port, sslmode, timezone)
+	dsn := fmt.Sprintf("host=%v user=%v password=%v dbname=%v port=%v sslmode=%v TimeZone=%v prefer_simple_protocol=true", host, user, password, dbname, port, sslmode, timezone)
```

Comment:
is there a reason for  adding `prefer_simple_protocol=true`
it's causing a problem

Reply:
Yeah, it was caching my statements, but you can remove it if it an error persists

Reply:
This thing you added broke our test pipeline bro

Reply:
Apologies for the inconvinience